scripts/ci/pre_commit/pre_commit_check_setup_installation.py

```diff
@@ -26,6 +26,10 @@
 from os.path import dirname
 from typing import Dict, List
 
+from rich import print as rprint
+from rich.console import Console
+from rich.table import Table
```

Comment:
I did not expected that we will adopt rich so fast 🚀

Reply:
I love rich, I am going to plug it in everywhere I can in Airflow :D